huaweicloud/services/acceptance/servicestage/resource_huaweicloud_servicestagev3_environment_test.go

```diff
@@ -0,0 +1,135 @@
+package servicestage
+
+import (
+	"fmt"
+	"regexp"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/config"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance/common"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/servicestage"
+)
+
+func getV3EnvFunc(conf *config.Config, state *terraform.ResourceState) (interface{}, error) {
+	client, err := conf.ServiceStageV2Client(acceptance.HW_REGION_NAME)
```

Comment:
```suggestion
	client, err := conf.ServiceStageV1Client(acceptance.HW_REGION_NAME)
```
Try to be consistent with the resource code.

Reply:
Should be NewServiceClient